### DISCUSSION TITLE
Audio not working with Ethos 1.5 firmware update

### DISCUSSION THREAD

**Discussion by wattscr** [FrSkyRC/ETHOS-Feedback-Community#3560] [General]
I updated Ethos 1.5 release and now I do not have any audio working for all of my special functions.

**Comment by wattscr** [comment]
> I updated Ethos 1.5 release and now I do not have any audio working for all of my special functions.

Resolved.   No longer an issue.

**Reply by Andreax20s** [reply]
I'd like to know how you solved it, I have the same problem

**Comment by Andreax20s** [comment]
come l'hai risolto?
Sto avendo lo stesso problema.
Thank you

**Reply by wattscr** [reply]
Yes...resolved.  Thanks

**Comment by wattscr** [comment]
Here is another source for the information ::
https://www.rcgroups.com/forums/showthread.php?3703807-FrSky-X20-X20S-X20HD-X20-Pro-Tandem-24-Channels-with-ETHOS/page1551#post50718883
